site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
+
+1. InitContainerRestoreHooks: These will add init containers into restored pods to perform any necessary setup before the application containers of the restored pod can start.
+1. ExecRestoreHooks: These can be used to execute custom commands or scripts in containers of a restored Kubernetes pod.
+
+Only InitContainerRestoreHooks is supported and support for ExectRestoreHooks is COMING SOON!
```

Comment:
Lets remove the all caps.

I know you marked Abbie's comment as resolved, but I'll note that #2855 does not discuss `ExecRestoreHooks` at all, while this PR does.

I'd rather not mention them in the docs if they're not present, and say they're coming soon in a blog post.

Reply:
#2855 does not mention `ExecRestoreHooks` because that is describing the current state of the restore API object.
As I mentioned in response to Abbie's comment. I don't want folks to think this will be the only supported restore hooks type while the design had two kinds.

Reply:
I'll remove the ALL-CAPS.

Reply:
I don't think I marked Abbie's comment as resolved. 😄 
I am OK w/ removing the coming soon bit. But, just want to make sure that folks don't feel we deviated from the design.